contracts/cw-core/src/tests.rs

```diff
@@ -144,7 +149,7 @@ fn test_instantiate_with_n_gov_modules(n: usize) {
 }
 
 #[test]
-#[should_panic(expected = "Execution would result in no governance modules being present.")]
+#[should_panic(expected = "Execution would result in no governance modules being active.")]
```

Comment:
How would you feel about changing this error to say "proposal modules" instead of "governance modules" while we're at it?